addons/payment/views/payment_form_templates.xml

```diff
@@ -427,8 +427,16 @@
                     class="btn-link alert-warning ps-2"
                     href="/web#action=payment.action_payment_provider"
                 >
-                    <strong><i class="oi oi-arrow-right"></i> View alternatives </strong>
+                    <strong><i class="oi oi-arrow-right"></i> Alternative Providers </strong>
                 </a>
+                <br/>
+                <a
+                    role="button"
+                    type="action"
+                    class="btn-link alert-warning ps-2"
+                    href="/web#action=payment.action_payment_method"
+                >
+                    <strong><i class="oi oi-arrow-right"></i> Payment Methods </strong></a>
             </div>
         </div>
         <!-- === Availability report (for debug) === -->
```

Comment:
You should remove debug part of comment :thinking: (also from other places where this and above report is called)

Reply:
Done